src/index.js

```diff
@@ -0,0 +1,77 @@
+import _ from 'lodash';
+import "./style.css";
+const formList = document.querySelector(".form");
+const addNew = document.querySelector(".input");
+const container = document.querySelector(".list");
+const clearAll = document.querySelector(".clear");
+const upDate = document.querySelector(".refresh");
+
+let list = JSON.parse(localStorage.getItem("list"));
```

Comment:
Todo items are not being stored in the local storage and hence the items are not being displayed. It's just a static page on my side.
#### As you can see in the screenshot
<img width="542" alt="Screen Shot 2022-05-08 at 8 12 27 AM" src="https://user-images.githubusercontent.com/83048208/167280131-7db4ebe2-0234-471d-9243-525a339405f6.png">